lxd/main_init.go

```diff
@@ -118,179 +105,32 @@ func (cmd *CmdInit) runAutoOrInteractive(c lxd.ContainerServer, backendsAvailabl
 			return err
 		}
 
-		storageBackend = cmd.Args.StorageBackend
-		storageLoopSize = cmd.Args.StorageCreateLoop
-		storageDevice = cmd.Args.StorageCreateDevice
-		storageDataset = cmd.Args.StorageDataset
-		networkAddress = cmd.Args.NetworkAddress
-		networkPort = cmd.Args.NetworkPort
-		trustPassword = cmd.Args.TrustPassword
-		storagePool = "default"
+		networking = &cmdInitNetworkingParams{
+			Address:       cmd.Args.NetworkAddress,
+			Port:          cmd.Args.NetworkPort,
+			TrustPassword: cmd.Args.TrustPassword,
+		}
 
 		// FIXME: Allow to configure multiple storage pools on auto init
 		// run if explicit arguments to do so are passed.
```

Comment:
No, because I don't think there's currently a way to express that with the current --auto command line flags (you can only specify the configuration for a single storage).

However considering that the --preseed option supports that, and it should eventually supersede --auto (which I guess could be dropped in LXD version 3), perhaps I can just drop this FIXME comment.